### PR TITLE
About page for 5.9.1

### PR DESCRIPTION
**Pull request by audrasjb**
Trac ticket: https://core.trac.wordpress.org/ticket/55173

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -39,6 +39,37 @@
...
+						/* translators: 1: WordPress version number, 2: plural number of bugs. */
+						_n(
+							'<strong>Version %1$s</strong> addressed %2$s bug.',
+							'<strong>Version %1$s</strong> addressed %2$s bugs.',
+							666
+						),
+						'5.9.1',
+						number_format_i18n( 666 )
+					);
+					?>
```
Comment (mukeshpanchal27) [inline src/wp-admin/about.php:57]: @audrasjb Can you please remove these PHP tags?

Comment (audrasjb) [reply src/wp-admin/about.php:57]: While I'd theoretically agree with you, this code comes from previous versions and I assume it's probably better to keep a minimal consistency here…

- 5.8 https://github.com/WordPress/wordpress-develop/blob/5.8/src/wp-admin/about.php
- 5.7 https://github.com/WordPress/wordpress-develop/blob/5.7/src/wp-admin/about.php
- 5.6 https://github.com/WordPress/wordpress-develop/blob/5.8/src/wp-admin/about.php
- 5.5 https://github.com/WordPress/wordpress-develop/blob/5.8/src/wp-admin/about.php

File: src/wp-admin/about.php [hunk 2]
```diff
@@ -39,6 +39,37 @@
...
+					<?php
+					printf(
+						/* translators: 1: WordPress version number, 2: plural number of bugs. */
+						_n(
+							'<strong>Version %1$s</strong> addressed %2$s bug.',
+							'<strong>Version %1$s</strong> addressed %2$s bugs.',
+							666
+						),
+						'5.9.1',
+						number_format_i18n( 666 )
```
Comment (hellofromtonya) [inline src/wp-admin/about.php]: @audrasjb is the number right? Just checking to make sure before it's committed.

Comment (audrasjb) [reply src/wp-admin/about.php]: no, that's a placeholder, I'll update it sooner :)

Comment (audrasjb) [reply src/wp-admin/about.php]: 82 is the correct number.

**Comment by audrasjb** [comment]
Committed to branch 5.9 in https://core.trac.wordpress.org/changeset/52785

**Pull request closed without merging**